src/test/librbd/mock/MockImageCtx.h

```diff
@@ -36,6 +36,10 @@ struct MockImageCtx {
     assert(s_instance != nullptr);
     return s_instance;
   }
+  void destroy() {
+    return;
```

Comment:
Still applies -- replace this definition w/ the ```MOCK_METHOD0``` macro to create a stub mock ```destroy``` method